meinberlin/static/embed.js

```diff
@@ -13,31 +15,48 @@ $(document).ready(function () {
     // adhocracy4.onReady($main)
   }
 
-  $(document).on('click', function (event) {
-    if (event.target.href && !event.target.target) {
+  $(document).on('click', 'a[href]', function (event) {
+    // NOTE: event.target.href is resolved against /embed/
+    var url = event.target.getAttribute('href')
+    var $link = $(event.target)
+    var embedTarget = $link.data('embedTarget')
+
+    if (
+      url &&
+      url[0] !== '#' &&
+      !event.target.target &&
+      embedTarget !== 'platform' &&
+      embedTarget !== 'external' &&
+      !$link.is('.rich-text *')
```

Comment:
I am not really happy with that. Maybe we should make a function out of that that returns a boolean. That would in the end just move the problem, but at least it looks better in this part of the code?